### PR TITLE
Add camera variants for Franka deformable lift environments

### PR DESCRIPTION
**Pull request by mmichelis**
# Description

Adds RGB camera variants of the existing Franka volume-deformable and cloth lift environments.

- Registers `Isaac-Lift-Soft-Franka-Camera` and `Isaac-Lift-Cloth-Franka-Camera`.
- Colocates the volume camera scene, environment, and shared visual observations with `franka_soft_env_cfg.py`.
- Colocates the cloth camera scene and environment with `franka_cloth_env_cfg.py` while reusing the identical camera and observation definitions.
- Routes target commands, previous actions, proprioception, and RGB observations to a CNN actor while providing the MLP critic with the same non-image observations plus privileged sampled deformable points.
- Uses the production camera configurations in the Franka deformable rendering regression tests.
- Documents the new task IDs and adds an `isaaclab_tasks` changelog fragment.

The camera variants inherit their actions, commands, rewards, events, terminations, and simulation behavior from the existing state-based environments. No new external dependencies are required.

## Type of change

- New feature (non-breaking change which adds functionality)
- Documentation update

## Videos

Each pair compares the same 128 x 128 camera rollout rendered with the Newton renderer and with the OVRTX renderer.

### Cloth camera environment

Production zero-action rollout using the native cloth action space.

| Newton renderer | OVRTX renderer |
| :---: | :---: |
| <video src="https://github.com/user-attachments/assets/8c5fe6a6-4917-4a02-a571-1841b2306b2e"></video> | <video src="https://github.com/user-attachments/assets/e0c024a9-a7d7-4f2f-b1ba-5d96b64cf07d"></video> |

### Volume deformable camera environment

Rollout driven by the existing Franka soft lift state machine.

| Newton renderer | OVRTX renderer |
| :---: | :---: |
| <video src="https://github.com/user-attachments/assets/9561d329-6cc0-4952-a289-639ccfd8d8cd"></video> | <video src="https://github.com/user-attachments/assets/5df38513-e352-4d62-9baf-81bd5d719196"></video> |

## Validation

- `./isaaclab.sh -f`
- `./isaaclab.sh -d` (warning-free Sphinx build)
- `./isaaclab.sh -p tools/update_environments_rst.py --check` (144 environments synchronized)
- `./isaaclab.sh -p scripts/environments/list_envs.py --show_presets`
- `./isaaclab.sh -p tools/changelog/cli.py check develop`
- Forbidden-import suite: 104 passed
- Soft RGB, cloth RGB, and soft depth Newton/Newton Warp golden rendering tests: passed
- Both registered tasks reset, stepped, exposed all four observation groups, and produced nonblank RGB observations
- One-iteration RSL-RL PPO smoke test: 24 steps completed with the CNN actor and MLP critic

The PPO smoke test covers integration, not policy convergence. Backend presets were resolved, but not every advertised physics and renderer combination was exercised end to end.

## Benchmarks

Runtime throughput from `scripts/benchmarks/runtime.py` (random actions) across the physics and renderer backends, measured at 128 environments and 100 frames with seed 42 on a single NVIDIA RTX 5090. Throughput is total environment steps per second (num_envs times frames divided by runtime), runtime is the 100-frame wall time, and startup is the launch-to-first-step time. The PhysX and OVRTX combination is unsupported and omitted, and the cloth task runs only on the Newton backend because it has no PhysX scene preset.

| Task | Physics | Renderer | Throughput (FPS) | Runtime (s) | Startup (s) | GPU mem (GB) |
| --- | --- | --- | ---: | ---: | ---: | ---: |
| Cloth | newton_mjwarp_vbd | isaacsim_rtx | 888.5 | 14.4 | 19.3 | 24.5 |
| Cloth | newton_mjwarp_vbd | newton_renderer | 1817.1 | 7.0 | 13.6 | 20.7 |
| Cloth | newton_mjwarp_vbd | ovrtx | 1028.3 | 12.4 | 19.2 | 24.8 |
| Soft | newton_mjwarp_vbd | isaacsim_rtx | 1148.3 | 11.1 | 15.7 | 11.2 |
| Soft | newton_mjwarp_vbd | newton_renderer | 3105.2 | 4.1 | 10.8 | 7.2 |
| Soft | newton_mjwarp_vbd | ovrtx | 1390.3 | 9.2 | 16.2 | 11.2 |
| Soft | physx | isaacsim_rtx | 1488.2 | 8.6 | 10.4 | 13.2 |
| Soft | physx | newton_renderer | 4032.8 | 3.2 | 13.7 | 11.5 |

## Checklist

- [x] I have read and understood the [contribution guidelines](https://isaac-sim.github.io/IsaacLab/main/source/refs/contributing.html)
- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added a changelog fragment under `source/<pkg>/changelog.d/` for every touched package (do **not** edit `CHANGELOG.rst` or bump `extension.toml`; CI handles that)
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/config/franka_soft/franka_soft_env_cfg.py [hunk 1]
```diff
@@ -331,6 +355,57 @@ def __post_init__(self) -> None:
...
+    class BaseImageCfg(ObsGroup):
+        image = ObsTerm(
+            func=env_mdp.image,
+            params={
+                "sensor_cfg": SceneEntityCfg("base_camera"),
+                "data_type": "rgb",
+                "normalize": True,
+                "permute": True,
+            },
+        )
```
[the remark this answers is not in the document]
Comment (mmichelis) [reply source/isaaclab_tasks/isaaclab_tasks/core/lift/config/franka_soft/franka_soft_env_cfg.py:404]: Thanks. This is intentional: ObservationGroupCfg already defaults to concatenate_terms=True and enable_corruption=False, which are the desired image semantics. The sibling groups override enable_corruption to True; BaseImageCfg keeps the defaults, matching the existing Dexsuite base-image groups. I will keep this as-is to avoid duplicating framework defaults.

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/config/franka_soft/agents/rsl_rl_ppo_cfg.py [hunk 2]
```diff
@@ -22,17 +43,34 @@ class FrankaDeformablePPORunnerCfg(RslRlOnPolicyRunnerCfg):
...
-        entropy_coef=0.006,
-        num_learning_epochs=5,
-        num_mini_batches=4,
-        learning_rate=1.0e-4,
-        schedule="adaptive",
-        gamma=0.98,
-        lam=0.95,
-        desired_kl=0.01,
-        max_grad_norm=1.0,
+    algorithm = ALGO_CFG
```
[the remark this answers is not in the document]
Comment (mmichelis) [reply source/isaaclab_tasks/isaaclab_tasks/core/lift/config/franka_soft/agents/rsl_rl_ppo_cfg.py:46]: Valid point. FrankaDeformablePPORunnerCfg.policy predates this camera change and remains supported by Isaac Lab RSL-RL compatibility bridge. Migrating the existing soft and cloth state-task runner changes established training configuration and requires dedicated training and checkpoint validation, so I will keep this PR scoped to the camera variants.

**Comment by mmichelis** [comment]
@greptile

**Review by kellyguo11 — APPROVED** [review]

**Review by huidongc — APPROVED** [review]

**Comment by kellyguo11** [comment]
hmm seems like the isaaclab_tasks failure might be related to this change

**Comment by mmichelis** [comment]
> hmm seems like the isaaclab_tasks failure might be related to this change

My agent thinks it's a flaky result because the rtx visualizer does not wait long enough with newton physics, so I'm trying out something with rerenders_on_reset. I'll see if that is sufficient. It is a bit odd that the rgb annotator does not immediately exist.

File: source/isaaclab_tasks/isaaclab_tasks/core/lift/config/franka_soft/franka_soft_env_cfg.py [hunk 3]
```diff
@@ -445,6 +520,18 @@ class FrankaSoftSceneCfg(PresetCfg):
...
 
 
+@configclass
+class FrankaSoftCameraSceneCfg(PresetCfg):
+    """Scene presets for visual Franka soft lifting."""
+
+    newton_mjwarp_vbd: _FrankaSoftCameraSceneCfg = _FrankaSoftCameraSceneCfg(
+        num_envs=128, env_spacing=2.5, replicate_physics=True
+    )
+    physx: _FrankaSoftCameraSceneCfg = _FrankaSoftCameraSceneCfg(num_envs=128, env_spacing=2.5, replicate_physics=False)
```
Comment (kellyguo11) [inline source/isaaclab_tasks/isaaclab_tasks/core/lift/config/franka_soft/franka_soft_env_cfg.py:530]:
```suggestion
    physx: _FrankaSoftCameraSceneCfg = _FrankaSoftCameraSceneCfg(num_envs=128, env_spacing=2.5, replicate_physics=False)
    isaacsim_physx = physx
```
